docs/families/WannaCry.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+# TODO - uncomment out the following. Do not change the name.
+parent: Ransomware Families
+
+# TODO - uncomment the below and change "Black Cat" to the new name. This will be shown in the menu.
+title: WannaCry
+
+# TODO - uncomment below and change "blackcat" to the new url. It should be all lower case and no spaces
+permalink: /families/wannacry/
+---
+
+| Category | Answer | References | 
+| ----------- | ----------- | | 
+|**Actors** | | |
+|First Observed | May 2017 | [^1] |
+|Threat Actors | North Korea | [^2]|
+|**Environment** | | |
+|Platforms | Windows | [^3]|
+|**Artifacts** | | |
+|Extensions | .wannacry .wcry .Wnry .wncry | [^4] [^5] [^6] [^7]|
+|Ransomware Notes | info.hta | [^8] |
+|Services It Disables | | |
+|Other Observables | Various YARA rules in existence | [^9] |
+|**Automation** | | |
+|Initial Access | Yes, EternalBlue| [^10] |
+|Privilege Escalation | No, not needed. Worm-like| |
+|Human Operated | No | |
+|Exfiltration | No | |
+|Propagation | Yes, worm-like | [^11] |
+
+
+[^1]: https://www.csoonline.com/article/3227906/wannacry-explained-a-perfect-ransomware-storm.html
+[^2]: https://www.cisa.gov/uscert/ncas/alerts/TA17-132A
+[^3]: https://www.csoonline.com/article/3227906/wannacry-explained-a-perfect-ransomware-storm.html
```

Comment:
We shouldn't re-use references.  So, we don't need to add this. We can just use ^1  for the Platforms instead of adding in a copy to the same URL.